### PR TITLE
Improved documentation about weights

### PR DESCRIPTION
**Pull request by brenthuisman**
In response to some questions at the IBRO-Ebrains workshop, here some minor updates in the documentation of connection and event generator weights.

### CODE REVIEW

**Review by schmitts — COMMENTED** [review]
LGTM

File: doc/python/interconnectivity.rst [hunk 1]
```diff
@@ -32,10 +32,11 @@ Interconnectivity
...
 
-        The weight delivered to the target synapse.
-        The weight is dimensionless, and its interpretation is specific to the type of the synapse target.
-        For example, the expsyn synapse interprets it as a conductance with units μS (micro-Siemens).
-
+        The weight delivered to the target synapse. It is up to the target mechanism to interpret this quantity.
+        For Arbor-supplied point processes, such as the ``expsyn`` synapse, a weight of ``1`` corresponds to an
+        increase in conductivity in the target mechanism of ``1`` μS (micro-Siemens).
+        For Arbor-supplied distributed processes, ``weight = 1`` induces ``1`` [S/cm2].
+
```
Comment (halfflat) [inline doc/python/interconnectivity.rst:39]: The only change I'd suggest here is to remove the mention of distributed processes, as these cannot be targets of connections.

File: doc/python/recipe.rst [hunk 2]
```diff
@@ -118,7 +118,10 @@ Event generator and schedules
 
     .. attribute:: weight
 
-        The weight of events to deliver.
+        The weight delivered to the target synapse. It is up to the target mechanism to interpret this quantity.
+        For Arbor-supplied point processes, such as the ``expsyn`` synapse, a weight of ``1`` corresponds to an
+        increase in conductivity in the target mechanism of ``1`` μS (micro-Siemens).
+        For Arbor-supplied distributed processes, ``weight = 1`` induces ``1`` [S/cm2].
```
Comment (halfflat) [inline doc/python/recipe.rst:124]: As above, re: distributed processes.

File: doc/tutorial/network_ring.rst [hunk 3]
```diff
@@ -72,7 +77,7 @@ are connecting the cells **(8)**, returning a configurable number of cells **(6)
 (``make_cable_cell()`` returns the cell above).
 
 Step **(8)** creates an :py:class:`arbor.connection` between consecutive cells. If a cell has gid ``gid``, the
-previous cell has a gid ``(gid-1)%self.ncells``. The connection has a weight of 0.1 μS and a delay of 5 ms.
+previous cell has a gid ``(gid-1)%self.ncells``. The connection has a weight of 0.01 μS and a delay of 5 ms.
```
Comment (halfflat) [inline doc/tutorial/network_ring.rst:80]: We shouldn't have the unit here on the weight.

File: python/example/network_ring.py [hunk 4]
```diff
@@ -87,15 +87,15 @@ def cell_kind(self, gid):
     # (8) Make a ring network. For each gid, provide a list of incoming connections.
     def connections_on(self, gid):
         src = (gid-1)%self.ncells
-        w = 0.01
+        w = 0.01 # μS on expsyn
```
Comment (halfflat) [inline python/example/network_ring.py:90]: Just to make it clear, I'd propose the comment `# 0.01 μS on expsyn` here and below.

**Review by halfflat — CHANGES_REQUESTED** [review]

**Review by halfflat — APPROVED** [review]
Thanks for the edits!

**Pull request merged**